nix/mk-shell.nix

```diff
@@ -59,17 +65,18 @@ in pkgs.mkShell {
         vector
         vty
       ]))
+    (workaround140774 pkgs.haskellPackages.ghcid)
+    (workaround140774 pkgs.haskellPackages.niv)
+    (workaround140774 pkgs.haskellPackages.ormolu)
     pkgs.apacheKafka # for nri-kafka
     pkgs.cabal-install
     pkgs.cachix
     pkgs.gnumake
-    (workaround140774 pkgs.haskellPackages.ghcid)
+    haskell-language-server
```

Comment:
should we include hls by default?  what if you don't use hls?  probably not a huge overhead though if you're forced to build it regardless of whether you actually use it

Reply:
I did consider not pushing it. Have been holding it for a while.
Since it builds just fine, I feel like keeping it around, but I'd be completely fine if this were to be dropped should it become troublesome during an update or something like that.

Reply:
(post-discussion) we're gonna leave it in.  shouldn't be too big a deal